tests/snmp/test_snmp_lldp.py

```diff
@@ -83,7 +83,8 @@ def test_snmp_lldp(ansible_adhoc, testbed, creds):
     for k, v in snmp_facts['snmp_interfaces'].items():
         if v.has_key("lldpRemManAddrIfSubtype") and \
            v.has_key("lldpRemManAddrIfId") and \
-           v.has_key("lldpRemManAddrOID"):
+           v.has_key("lldpRemManAddrOID") and \
+           v['name'] != 'eth0':
```

Comment:
why do we skip eth0 in line 78? Can we also fix this issue by not skip eth0 in line 78?

Reply:
I think the original intention to skip 'eth0' in line 78 is to rule out the LLDP neighbor connected to the mgmt interface. I am kind of agree with the original intention since we only care about the neighbors from the front panel ports.  If you think we shall also take mgmt interface into account then the fix will be change line 78 to not skip eth0.  what's your view?

Reply:
ok. maybe eth0 has multiple neighbors and will have multiple rem man address entries which could violate the validation.